src/FirstSteps/IAdventure.cs

```diff
@@ -0,0 +1,13 @@
+﻿using FirstSteps.RPG.Heroes;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace FirstSteps
+{
+    public interface IAdventure
+    {
+        public void Enter(Hero hero);
+        // u mean public void Enter(Hero hero, string command) ? 
```

Comment:
I meant `string Command { get; }`. It will be responsible for storing info about command text. In the future task, you will use it to display commands for the user. 
For example, the command for the `Forest` should store text that you use here in `case` option:
```csharp
case "forest":
    _forest.Enter(_hero);
    break;
```